File: 62-unique-paths/62-unique-paths.py
class Solution:
    def uniquePaths(self, m: int, n: int) -> int:
        
        
        d =[ [1]*n for _ in range(m)]
        
        for r in range(1,m):
            for c in range(1,n):
                d[r][c] = d[r-1][c] + d[r][c-1]
        return d[m-1][n-1]
       

File: 62-unique-paths/62-unique-paths.cpp
class Solution {
public:
    int uniquePaths(int R, int C) {
        
        
        vector<int> cdp(C ,1);
        for(int r =1; r < R ; r++){
            for(int c =1 ; c < C ; c++){ 
                cdp[c] += cdp[c-1] ; 
            }}
        return cdp[C-1] ; 



        
       
        
    }
};